core/src/main/java/org/apache/brooklyn/core/mgmt/persist/DeserializingClassRenamesProvider.java

```diff
@@ -37,7 +37,8 @@
 public class DeserializingClassRenamesProvider {
 
     public static final String DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH = "classpath://org/apache/brooklyn/core/mgmt/persist/deserializingClassRenames.properties";
-    
+    public static final String KARAF_DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH = "file:etc/deserializingClassRenames.properties";
+
```

Comment:
I wasn't sure about the name, hence my note in the description.

@m4rkmckenna What about `org.apache.brooklyn.deserializeClassRename.properties` ?